include/deal.II/lac/sparse_direct.h

```diff
@@ -634,12 +665,18 @@ class SparseDirectMUMPS : public EnableObserverPointer
   types::global_dof_index n;
 
   /**
-   * The number of non-zero entries in the matrix.
+   * Number of non-zero entries in the matrix.
    */
   types::mumps_nnz nnz;
 
   /**
-   * This function hands over to MUMPS the system's <tt>matrix</tt>.
+   * IndexSet storing the locally owned rows of the matrix.
+   */
+  IndexSet row_range;
```

Comment:
Would you mind calling this variable `locally_owned_rows` to make it clearer what the index set actually represents? The comment makes that clear, but the variable name does not.